src/content/en/2021/resource-hints.md

```diff
@@ -1,20 +1,648 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: Resource Hints
-description: TODO
-authors: []
-reviewers: []
-analysts: []
-editors: []
+description: Resource Hints chapter of the 2021 Web Almanac covering adoption of resource hints, their uses, bad practices, and their impact on performance.
+authors: [kevinfarrugia]
+reviewers: [siakaramalegos, tunetheweb, andydavies, samarpanda]
```

Comment:
Adding @westonruter as a reviewer.

Could you add his contributor info to `2021.json` as well?

```json

    "westonruter": {
      "name": "Weston Ruter",
      "teams": [
        "reviewers"
      ],
      "avatar_url": "https://avatars.githubusercontent.com/u/134745?v=4&s=200",
      "website": "https://weston.ruter.net/",
      "github": "westonruter",
      "twitter": "westonruter"
    },
```

```suggestion
reviewers: [siakaramalegos, tunetheweb, andydavies, samarpanda, westonruter]
```

Reply:
Added